src/bundle/Controller/ContentViewController.php

```diff
@@ -0,0 +1,111 @@
+ */
+
+namespace EzSystems\HybridPlatformUiBundle\Controller;
+
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use eZ\Publish\Core\MVC\Symfony\Controller\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class ContentViewController extends Controller
+{
+    protected $defaultSortFields = [
+        Location::SORT_FIELD_NAME => ['key' => 'sort.name', 'default' => 'Content name'],
+        Location::SORT_FIELD_PRIORITY => ['key' => 'sort.priority', 'default' => 'Priority'],
+        Location::SORT_FIELD_MODIFIED => ['key' => 'sort.modified', 'default' => 'Modification date'],
+        Location::SORT_FIELD_PUBLISHED => ['key' => 'sort.published', 'default' => 'Publication date']
+    ];
+
+    protected $otherSortFields = [
+        Location::SORT_FIELD_PATH => ['key' => 'sort.path', 'default' => 'Location path'],
+        Location::SORT_FIELD_CLASS_IDENTIFIER => ['key' => 'sort.content.type.identifier', 'default' => 'Content type identifier'],
+        Location::SORT_FIELD_SECTION => ['key' => 'sort.section', 'default' => 'Section'],
+        Location::SORT_FIELD_DEPTH => ['key' => 'sort.depth', 'default' => 'Location depth'],
+        Location::SORT_FIELD_CLASS_NAME => ['key' => 'sort.content.type.name', 'default' => 'Content type name']
+    ];
+
+    public function detailsTabAction(ContentView $view)
+    {
+        $repository = $this->getRepository();
+
+        $versionInfo = $view->getContent()->getVersionInfo();
+        $contentInfo = $versionInfo->getContentInfo();
+
+        $sectionService = $this->getRepository()->getSectionService();
+        $section = $sectionService->loadSection($contentInfo->sectionId);
+
+        $userService = $this->getRepository()->getUserService();
+
+        $creator = $repository->sudo(function () use ($contentInfo, $userService) {
+            try {
+                return $userService->loadUser($contentInfo->ownerId);
+            } catch (NotFoundException $e) {
+                return null;
+            }
+        });
+
+        $lastContributor = $repository->sudo(function () use ($versionInfo, $userService) {
+            try {
+                return $userService->loadUser($versionInfo->creatorId);
+            } catch (NotFoundException $e) {
+                return null;
+            }
+        });
+
+        $location = $view->getLocation();
+
+        $view->addParameters([
+            'section' => $section,
+            'contentInfo' => $contentInfo,
+            'versionInfo' => $versionInfo,
+            'creator' => $creator,
+            'lastContributor' => $lastContributor,
+            'translations' => $this->getTranslations($versionInfo),
+            'ordering' => [
+                'sortFields' => $this->getSortFields($location->sortField),
```

Comment:
You can use `$view->getLocation->sortField`, since you don't use `$location` anywhere else.